docs/modules/ROOT/pages/advanced-how-tos/managing-multiple-versions.adoc

```diff
@@ -161,6 +161,27 @@ event == "push" && target_branch == "v1.0.0"
 
 * The `appstudio.openshift.io/application` and `appstudio.openshift.io/component` labels must be adjusted to specify the right Application and Component respectively. Failing to do this will cause builds of the pipeline to be associated with the wrong application or component.
 
+=== Create ImageRepositories for each component
+
+This must be done for each component to authenticate with the output-registry when pushing built images.
```

Comment:
Is this really needed for all cases? What about the case where all the different versions push into the same image repo? This is what the examples were really showing so far.

Reply:
Yes, we can have one image repository configured for many components. In general you should proceed with single component as usual, and omit any image repository actions for others except setting Soec.ContainerImage to already existing image repository.
Note, that image repository is bound to the first component and removing it will leave all other components without image repository at all. To prevent that, one should add each component into ownerReference of the ImageRepository CR.